CalculatorApp/Controller/ViewController.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  ViewController.swift
+//  CalculatorApp
+//
+//  Created by Memory Mhou on 04/03/2024.
+//
+
+import UIKit
+
+
+class ViewController: UIViewController {
+    
+ @IBOutlet weak var OutPutLabel: UILabel!
+    
+    var runningNumber = ""
+    var leftValue = ""
+    var rightValue = ""
+    var currentOperation: CalculatorModel.Operation?
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        OutPutLabel.text = "0"
+    }
+    
```

Comment:
remove spacing, you should always have oneline spacing

Reply:
I have made the requested changes.